src/components/AutoSearch.tsx

```diff
@@ -14,12 +15,14 @@ interface AutoSearchProps {
 }
 
 export const AutoSearch: React.FC<AutoSearchProps> = ({
+  clearSearchField,
   onInputChange,
   searchResults,
   setValueFunction,
   label,
   groupBy,
 }) => {
+  const [value, setValue] = useState({ title: '' });
```

Comment:
`const [title, setTitle] = useState('');`?

Reply:
`The value must have reference equality with the option in order to be selected.`
https://material-ui.com/api/autocomplete/

Reply:
vil ikke det fungere hvis vi fjerner custom `getOptionLabel`-prop?

Reply:
akkurat nå brukes autoSearch kun til å slå opp i lister som består av objekt med titler og da må vi spesifisere at det er det som skal være label'en. I fremtiden kan vi tilpasse autoSearch til å kunne brukes med hva som helst, men synes det er greit å la det være for nå.